test/options.h

```diff
@@ -43,6 +45,7 @@ class options {
     std::set<std::string> _options;
     std::string _which_test;
     const bool _quiet;
+    const bool _help;
```

Comment:
If you want to further cleanup this later ... I would like that the variable names are `mQuiet`, `mHelp`, etc..

Reply:
I'll take a look, for another PR.

Reply:
See #1720.